frontends/web/src/routes/bitboxbase/bitboxbase.tsx

```diff
@@ -225,9 +249,9 @@ class BitBoxBase extends Component<Props, State> {
         this.setState({ password });
     }
 
-    private submitChangePassword = (event: Event) => {
+    private submitChangePasswordSetup = (event: Event) => {
         event.preventDefault();
-        apiPost(this.apiPrefix() + '/user-change-password', {username: 'admin', newPassword: this.state.password})
+        apiPost(this.apiPrefix() + '/user-change-password', {username: 'admin', password: defaultPassword, newPassword: this.state.password})
```

Comment:
defaultPassword is coming from https://github.com/digitalbitbox/bitbox-wallet-app/blob/7e6ffe14af91bdb994a0b7782cd82c7d1631106c/frontends/web/src/routes/bitboxbase/bitboxbase.tsx#L49 is that correct? it never changes

Reply:
I'm sure now it makes sense somehow. just ignore my comment :)